client/src/components/EmailModal/EmailModal.component.tsx

```diff
@@ -0,0 +1,146 @@
+  onConfirm: (data: Enquiry) => Promise<void>
+  agency: Agency
+}
+
+export const EmailModal = ({
+  isOpen,
+  onClose,
+  onConfirm,
+  agency,
+}: EmailModalProps): JSX.Element => {
+  const { register, handleSubmit, reset, formState } = useForm()
+  const { errors: formErrors } = formState
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+
+  const onSubmit: SubmitHandler<Enquiry> = async (data) => {
+    setIsLoading(true)
+    await onConfirm(data)
+    setIsLoading(false)
+    reset()
+    onClose()
+  }
+
+  return (
+    <Modal isOpen={isOpen} onClose={onClose} size="xl">
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <ModalOverlay />
+        <ModalContent w="660px">
+          <ModalHeader>
+            <Text textStyle="h2" color="secondary.700">
+              {`${agency.shortname.toUpperCase()} Enquiry Form`}
+            </Text>
+          </ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <VStack align="left" spacing={0}>
+              <Text>
+                {`This enquiry form will be sent to ${agency.longname}. Please do
+                note that we would take at least 3 working days to process your
+                enquiry. Thank you.`}
+              </Text>
+              <Box h={4} />
+              <Text textStyle="subhead-1" color="secondary.700">
+                Question Title
+              </Text>
+              <Box h={3} />
+              <Input
+                focusBorderColor="primary.500"
+                errorBorderColor="error.500"
+                isInvalid={formErrors.questionTitle}
+                {...register('questionTitle', {
+                  required: true,
+                })}
+              />
+              {formErrors.questionTitle && errorLabel('This field is required')}
+              <Box h={4} />
+              <Text textStyle="subhead-1" color="secondary.700">
+                Description
+              </Text>
+              <Box h={3} />
+              <Textarea
+                focusBorderColor="primary.500"
+                errorBorderColor="error.500"
+                isInvalid={formErrors.description}
+                h="144px"
+                {...register('description', {
+                  required: true,
+                })}
+              />
+              {formErrors.description && errorLabel('This field is required')}
+              {/* <Box h={4} />
+              <Text textStyle="subhead-1" color="secondary.700">
+                Sender email
+              </Text>
+              <Text textStyle="body-2" color="secondary.400">
+                Please fill in a valid email so that we can get back to you
+              </Text>
+              <Box h={3} />
+              <Input
+                focusBorderColor="primary.500"
+                errorBorderColor="error.500"
+                isInvalid={formErrors.senderEmail}
+                placeholder="jony_tan@gmail.com"
```

Comment:
Avoid having an e-mail address from an actual domain here. There is no apparent consensus as to what should be an appropriate placeholder for e-mail fields; will leave it at your discretion to have something sensible.

@khaleedah in case she wants to weigh in

Reply:
changing to email